### DISCUSSION TITLE
Guidance on Themes (iOS, MacOs, ...)

### DISCUSSION THREAD

**Discussion by stefandevo** [AvaloniaUI/Avalonia#12994] [Q&A]
Are there any plans to have out of the box support for iOS/macOS (Cupertino) styles?
If not, is there any way to start with such themes? I checked the Material theme, and it seems a lot of work :-)
What would be the best to start? Can we "steal" from Flutter implementations easily?

**Comment by MikeCodesDotNET** [comment]
There are no plans from the core team, but I started a theme many years ago, and it wasn't very difficult to get started. 

![image](https://github.com/AvaloniaUI/Avalonia/assets/552074/61c052c4-1309-4930-b559-d017d77cad07)

**Reply by stefandevo** [reply]
Do you still have the code around ... maybe I can use that as a start ?

**Reply by MikeCodesDotNET** [reply]
I probably have the code somewhere, but it's for an older version of Avalonia. The screenshot is an afternoon or two of work, so shouldn't take long to recreate.
